RELEASE-NOTES.md

```diff
@@ -249,6 +250,7 @@ Patricio Benavente <patbenavente@gmail.com>
 Raymond Roberts
 Rodrigo Benenson <rodrigo.benenson@gmail.com>
 Sergei Lebedev <superbobry@gmail.com>
+Shashank Shekhar <shashank.f1@gmail.com>
```

Comment:
Can you add yourself instead to a new section `Contributors` for the upcoming 3.3 release?

Reply:
Done